lib/ecstatic.js

```diff
@@ -80,6 +84,9 @@ var ecstatic = module.exports = function (dir, options) {
       return status[403](res, next);
     }
 
+    if (req.method === 'OPTIONS') {
+      return res.end()
+    }
```

Comment:
Might have to feature flag this. Wondering if it's worth doing now or later (I don't mind a major semver bump)